src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -369,58 +381,51 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
     const { sheetId, col, row } = target;
     const targetCell = this.getters.getEvaluatedCell(target);
 
-    if (clipboardOption?.pasteOption !== "onlyValue") {
-      const targetBorders = this.getters.getCellBorder(target);
-      const originBorders = origin.border;
-      const border = {
-        top: targetBorders?.top || originBorders?.top,
-        bottom: targetBorders?.bottom || originBorders?.bottom,
-        left: targetBorders?.left || originBorders?.left,
-        right: targetBorders?.right || originBorders?.right,
-      };
-      this.dispatch("SET_BORDER", { sheetId, col, row, border });
-    }
-    if (origin.cell) {
-      if (clipboardOption?.pasteOption === "onlyFormat") {
-        this.dispatch("UPDATE_CELL", {
-          ...target,
-          style: origin.cell.style,
-          format: origin.evaluatedCell.format,
-        });
-        return;
-      }
+    if (clipboardOption?.pasteOption === "onlyValue") {
+      const locale = this.getters.getLocale();
+      const content = formatValue(origin.evaluatedCell.value ?? "", { locale });
```

Comment:
null coalescing is useless. `value` is never undefined nor null
```suggestion
      const content = formatValue(origin.evaluatedCell.value, { locale });
```